resources/idlharness.js

```diff
@@ -1251,6 +1184,110 @@ IdlInterface.prototype.test_self = function()
 };
 
 //@}
+IdlInterface.prototype.test_immutable_prototype = function(type, obj)
+//@{
+{
+    if (typeof Object.setPrototypeOf !== "function") {
```

Comment:
Well, sort of. The broader question here is how do we handle tests we're unable to run because of lack of support for X.

I'm of the opinion we should throw in such cases, not silently avoid running a bunch of tests.